### PR TITLE
Add Chinese version

### PR DESCRIPTION
**Pull request by InCerryGit**

### CODE REVIEW

File: doc/.NETMemoryPerformanceAnalysis.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,1419 @@
...
+
+我在上面提到过几次钉住。大多数人都知道钉住是什么 - 它向GC表示一个对象不能被移动。但从GC的角度来看，钉住的意义是什么呢？由于GC不能移动这些被钉住的对象，它使被钉住的对象之前的死角变成了一个自由对象，这个自由对象可以用来容纳年轻一代的生存者。但这里有一个问题 - 正如我们从上面的代际讨论中看到的，如果我们简单地将这些被钉住的对象提升到老一代，就意味着这些自由空间也是老一代的一部分，要用它们来容纳年轻一代的幸存者，唯一的办法就是我们真的对年轻一代做一次GC（否则我们甚至没有 "年轻一代的幸存者"）。然而，如果我们能在gen0中保留这些自由空间，它们就可以被用户分配使用。这就是为什么GC有一个叫做降代的功能，我们将把这些被钉住的对象降代到gen0，这意味着它们之间的空闲空间将是gen0的一部分，当用户代码分配时，我们可以立即使用它们。
+
+图5 - 降代（我从一张旧的幻灯片上取下来的，所以这看起来与之前的片段图片有些不同。）
+
+<img src=".\demotion.jpg" alt="demotion" style="zoom:80%;" />
+
+由于gen0分配可以发生在这些自由空间中，这意味着它们将消耗gen0预算而不增加gen0的大小（除非自由空间不能满足所有的gen0预算，在这种情况下它将需要增长gen0）。
+
+然而，GC 不会无条件地降代，因为我们不想在 gen0 中留下许多固定对象，这意味着我们必须在每次 GC 中再次查看它们，可能会有很多次 GC（因为它们是 gen0 的一部分，每当我们执行 gen0 GC 我们需要查看它们）。 这意味着如果您遇到严重的固定情况，它仍然会导致 gen2 中的碎片问题。 同样，GC 确实有机制来应对这些情况。 但是如果你想对 GC 施加更少的压力，你可以从用户的 POV 中遵循这个规则—
```
Comment (Maoni0) [inline doc/.NETMemoryPerformanceAnalysis.zh-CN.md:548]: POV could be translated to Chinese right? you could say something like "從用戶的角度來説，你可以遵循這個規則“

File: doc/.NETMemoryPerformanceAnalysis.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,1419 @@
...
+  
+  目前，在较小的机器上（即内存小于80GiB），默认情况下GC将90%视为高内存负荷。在有更多内存的机器上，这是在90%到97%之间。这个阈值可以通过[COMPlus_GCHighMemPercent](https://docs.microsoft.com/zh-cn/dotnet/core/run-time-config/garbage-collector#high-memory-percent)环境变量（或者从.NET 5开始在runtimeconfig.json中配置System.GC.HighMemoryPercent）来调整。你想调整这个的主要原因是为了控制堆的大小。例如，在一台有64GB内存的机器上，对于主要的主导进程，当有10%的内存可用时，GC开始反应是合理的。但是对于较小的进程（例如，如果一个进程只消耗1GB的内存），GC可以在<10%的可用内存下舒适地运行，所以你可能想对这些进程设置得更高。另一方面，如果你想让较大的进程拥有较小的堆大小（即使机器上有大量可用的物理内存），把这个值调低将是一个有效的方法，让GC更快地做出反应，压缩堆的大小。
+  
+  对于在容器中运行的进程，GC会根据容器的限制来考虑物理内存。
+  
+  [本节](#计算出gen2-GC的工作量)描述了如何找出每个GC观察到的内存负载。
+  
+#### **了解GC是如何被触发的**
+
+  到目前为止，我们用GC来指代组件。下面我将用GC来指代组件，或者指代一个或多个在堆上进行内存回收的集合行为，即GC或GCs。
```
Comment (Maoni0) [inline doc/.NETMemoryPerformanceAnalysis.zh-CN.md:300]: "即GC或GCs" probably want to say "即一個或多個GC".

**Comment by Maoni0** [comment]
I'll do more reading tomorrow but so far the translation looks really good to me - I only noticed minor things. thank you so much for doing this! 👍👍

File: doc/.NETMemoryPerformanceAnalysis.zh-CN.md (hunk 1 above)
Comment (InCerryGit) [reply doc/.NETMemoryPerformanceAnalysis.zh-CN.md]: Ok, I'll fix it now. Thanks!

File: doc/.NETMemoryPerformanceAnalysis.zh-CN.md (hunk 2 above)
Comment (InCerryGit) [reply doc/.NETMemoryPerformanceAnalysis.zh-CN.md]: Ok, I'll fix it now. Thanks!

**Comment by Maoni0** [comment]
thanks so much!

**Comment by Maoni0** [comment]
(sorry, accidently closed it)

**Comment by InCerryGit** [comment]
Thanks to [@晓青](https://www.bluetian.net/)、[@贾佬](https://www.newbe.pro/)、[@黑洞](https://www.cnblogs.com/blurhkh/)、[@晓晨](https://www.cnblogs.com/stulzq/)、[@一线码农](https://www.cnblogs.com/huangxincheng/) @pengweiqhca for proofreading and feedback.

**Pull request merged**